8_reference/80_relay_api.txt

```diff
@@ -0,0 +1,102 @@
+=== Rudder relay API
+
+The `rudder-server-relay` package provides an HTTP API.
+It is available on simple relays and root servers.
+It is an internal API, not exposed to users, and used
+to provide various Rudder features.
+
+==== Remote Run
+
+The remote run API is available at `https://relay/rudder/relay-api/remote-run`.
+It allows triggering a run on nodes (like with the `rudder remote run` command).
+
+===== Description
+
+The remote run API applies to all nodes that are below the target relay server, which means:
+
+* All nodes directly connected to the server
+* All the relays that have the target node as policy server, and all nodes that are below them
+
+In particular, it does not act on the target relay itself (except for the root server which is its own policy server).
+
+There are different methods, whether you want to trigger all nodes, and only a part of them.
+
+===== Security
+
+The remote run calls are not authenticated, but restricted to:
+
+* Local calls
+* The target's policy server
```

Comment:
the target is ambiguous, prefer the term node for the remote run target, and relay for the api target